packages/core-js/test/utils/finders.spec.js

```diff
@@ -0,0 +1,30 @@
+const { equal } = require('assert/strict');
+const { findContractDependencies, findFunctionSelectors } = require('../../utils/ast/finders');
+const asts = require('../fixtures/asts.json');
+
+describe('utils/ast/comparator.js compareStorageStructs', function () {
```

Comment:
added some basic tests. We can populate it later with more complex conditions and all the functions exported. The task is already in our backlog